packages/webpack-cli/lib/utils/logger.js

```diff
@@ -1,9 +1,10 @@
-const { red, cyan, yellow } = require('colorette');
+const { red, cyan, yellow, green } = require('colorette');
 
 module.exports = {
     error: (val) => console.error(`[webpack-cli] ${red(val)}`),
     warn: (val) => console.warn(`[webpack-cli] ${yellow(val)}`),
     info: (val) => console.info(`[webpack-cli] ${cyan(val)}`),
+    success: (val) => console.log(`[webpack-cli] ${green(val)}`),
     log: (val) => console.log(`[webpack-cli] ${val}`),
-    help: (val) => console.log(val),
+    rawLog: (val) => console.log(val),
```

Comment:
I don't think we should avoid `[webpack-cli]` prefix here, because in theory, other things can logs here too

Reply:
So we can use just `log`